bin/pycbc_submit_dax

```diff
@@ -46,11 +45,6 @@ eval set -- "$GETOPT_CMD"
 
 while true ; do
   case "$1" in
-    -d|--dax)
```

Comment:
This is still present in the `--help` message

Reply:
Also can we now remove `--local-dir` if that's handled up a bit? .... I'm thinking that submit_dax could be removed reasonably quickly now.

Reply:
Ah, right, I'll take out the option and clean up the help.